tools/wptserve/tests/functional/test_cookies.py

```diff
@@ -47,18 +47,20 @@ def handler(request, response):
 
         self.assertEqual(parts["name"], "")
```

Comment:
Wouldn't these have to change as well?

Reply:
No, `resp` is actually the return value from `urllib.open` and is only used in these functional tests.